### DISCUSSION TITLE
Converging on style in organize-imports IntelliJ and Scala Center

### DISCUSSION THREAD

**Discussion by sideeffffect** [liancheng/scalafix-organize-imports#159] [General]
@liancheng  @nafg @pavelfatin

Has there been a discussion between organize-imports developers, IntelliJ and Scala Center on the convergence of how to sort imports in Scala? What are the outcomes?
There has been a new release of `organize-imports`
https://github.com/liancheng/scalafix-organize-imports/releases/tag/v0.5.0
It changes the default sort order over to that of IntelliJ, which is awesome! :tada: 

However, `groupedImports` is set to `Explode`. The arguments for using `Explode` are _very_ good. But IntelliJ does `Merge` by default, which causes grief, because now they're in conflict and don't converge when a developer uses both (IntelliJ and organize-imports) tools. Will IntelliJ change to `Explode` by default to solve this issue?

**Comment by liancheng** [comment]
@sideeffffect, thanks for starting this thread. According to the discussion in PR #154, the conversation between IntelliJ and Scala Center happened in a private Slack channel. I'm curious to learn the outcome.

**Reply by sideeffffect** [reply]
Oh, I didn't realize that. Maybe we should add some people from Scala Center. Perhaps @julienrf would know. Julien were you part of the conversation with JetBrains about imports in Scala? Or do you know who was?
